queries.sql

```diff
@@ -15,3 +16,34 @@ SELECT * from animals WHERE neutered = 'true';
 SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Day2
+
+BEGIN;
+UPDATE animals SET species = 'unspecifie';
```

Comment:
- [ ] Please fix the spelling. ``'unspecified'`` instead of ``'unspecifie'`` :+1: